### PR TITLE
Update to latest o-colors and o-typography

### PR DESCRIPTION
**Pull request by JakeChampion**
Old:
old single
old multiple
New:
new single
new multiple

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +79,24 @@ No _tabs_ will be shown, and _panel bodies_ will all be shown one below the othe
 
 _Tabs_ will be shown (if declared in markup) and will be functional. Only the _panel body_ for the selected _tab_ will be shown.
 
+## Troubleshooting
+
+### No tabs are showing, all panels are being shown
+That is to be expected for core experience.
+
+## Migration guide
+_Migration guides are very important! Always include one for major releases. To create a codeblock that has diff highligting, use three backticks followed by the word diff_
```
Comment (rowanmanning) [inline README.md]: Supposed to be left in?

File: bower.json [hunk 2]
```diff
@@ -8,8 +8,7 @@
     ".travis.yml"
   ],
   "dependencies": {
-    "o-colors": ">=2.5 <4",
-    "o-typography": "^3.0.0",
-    "o-fonts": "~2.1.0"
+    "o-colors": "^4",
+    "o-typography": "^5.0.0"
```
Comment (rowanmanning) [inline bower.json]: Why the difference in semver notation? Like `^M` vs `^M.m.p`

File: src/scss/panel.scss [hunk 3]
```diff
@@ -15,11 +15,11 @@
 
 /**
  * Apply to an aside header block
- * @requires {mixin} oTypographyHeading2
+ * @requires {mixin} oTypographyDisplay
  * @requires {function} oColorsGetColorFor
  */
 @mixin oAsidePanelHeader {
-	@include oTypographyHeading2;
+	@include oTypographyDisplay($scale: 3, $line-height: 31px);
```
Comment (rowanmanning) [inline src/scss/panel.scss]: You _can_ use `oTypographyHeadingLevel2` here, but not sure whether you already discussed this with design/adam :)

Comment (JakeChampion) [inline src/scss/panel.scss]: Line-height needs to be customised which means we can not use `oTypographyHeadingLevel2`

File: bower.json (hunk 2 above)
Comment (JakeChampion) [inline bower.json]: No reason, will make them align

**Comment by JakeChampion** [comment]
Waiting on o-tabs work to be completed

**Comment by alicebartlett** [comment]
Good to see the end of Benton!

**Pull request merged**